Dockerfile

```diff
@@ -52,7 +52,7 @@ ENV CRIU_VERSION 3.11
 RUN mkdir -p /usr/src/criu \
 	&& curl -sSL https://github.com/checkpoint-restore/criu/archive/v${CRIU_VERSION}.tar.gz | tar -C /usr/src/criu/ -xz --strip-components=1 \
 	&& cd /usr/src/criu \
-	&& make \
+	&& make -j $(nproc) \
```

Comment:
Curious, IIUC, this will _limit_ the number of jobs at once (as the default is "infinite"), correct?

```
  -j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.
```

Reply:
Yes, the default value, "infinite", can quickly take over all available memory on a large build. Although `criu` is not very large, `-j $(nproc)` works well, and it is the approach that is used in the [Dockerfile template](https://github.com/checkpoint-restore/criu/blob/master/scripts/build/Dockerfile.tmpl) for travis-ci.

Reply:
Thanks for explaining 👍